mawabot/cogs/text.py

```diff
@@ -82,6 +83,36 @@ def __init__(self, bot):
 
         await ctx.message.edit(content='\n'.join(lines))
 
+    @commands.command()
+    async def rev(self, ctx, *, text: str):
+        ''' Reverses the text given '''
+
+        await ctx.message.edit(content=text[::-1])
+
+    @commands.command()
+    async def kerrhau(self, ctx, *text: str):
+        ''' "kerrhau"-ifys the given text for meme emphasis '''
+
+        text = list(text)
+        words = []
+
+        while text:
+            rand_num = randint(1, 3)
+            word = []
+
+            for _ in range(rand_num):
```

Comment:
I would either rename `rand_num` to like `word_count` or just place `randint(1, 3)` directly in `range()`